internal/pkg/scaffold/ansible/dockerfilehybrid.go

```diff
@@ -45,14 +45,20 @@ func (d *DockerfileHybrid) GetInput() (input.Input, error) {
 	return d.Input, nil
 }
 
-const dockerFileHybridAnsibleTmpl = `FROM ansible/ansible-runner
+const dockerFileHybridAnsibleTmpl = `FROM ansible/ansible-runner:1.3.2
```

Comment:
1.3.3 breaks molecule

Reply:
Can we not keep it at 1.2.X? for now?

Reply:
Sure, this was just the most recent ansible-runner version that worked with molecule.